content/push/publish.textile

```diff
@@ -0,0 +1,776 @@
+  }
+}
+
+rest.push.admin.publish(recipient, notification)
+```
+
+```[jsall]
+var recipient = {
+  transportType: 'apns',
+  deviceToken: 'xxxxxxxxxx'
+};
+var notification = {
+  notification: {
+    title: 'Hello from Ably!',
+    body: 'Example push notification from Ably.'
+  }
+};
+
+rest.push.admin.publish(recipient, notification);
+```
+
+```[objc]
+ARTpushRecipient *recipient = @{
+    @"transportType": @"apns",
+    @"deviceToken": @"XXXXXXXX"
+};
+
+ARTJsonObject *data = @{
+    @"notification": @{
+        @"title": @"Hello from Ably!",
+        @"body": @"Example push notification from Ably."
+    },
+    @"data": @{
+        @"foo": @"bar",
+        @"baz": @"qux"
+    }
+};
+[rest.push.admin publish:recipient data:data callback:^(ARTErrorInfo *error)
+```
+
+```[swift]
+let recipient: [String: Any] = [
+    "transportType": "apns",
+    "deviceToken": "XXXXXXXX"
+]
+
+let data: [String: Any] = [
+    "notification": [
+        "title": "Hello from Ably!",
+        "body": "Example push notification from Ably."
+    ],
+    "data": [
+        "foo": "bar",
+        "baz": "qux"
+    ]
+]
+rest.push.admin.publish(recipient, data: data)
+```
+
+```[java]
+Message message = new Message("example", "rest data");
+message.extras = io.ably.lib.util.JsonUtils.object()
+    .add("push", io.ably.lib.util.JsonUtils.object()
+        .add("notification", io.ably.lib.util.JsonUtils.object()
+            .add("title", "Hello from Ably!")
+            .add("body", "Example push notification from Ably."))
+        .add("data", io.ably.lib.util.JsonUtils.object()
+            .add("foo", "bar")
+            .add("baz", "qux")));
+
+rest.push.admin.publish(arrayOf(Param("transportType", "apns"), Param("deviceToken", deviceToken)), message);
+```
+
+```[python]
+recipient = {'transportType': 'apns', 'deviceToken': 'XXXXXXX'}
+message = {
+  'push': {
+    'notification': {
+      'title': 'Hello from Ably!',
+      'body': 'Example push notification from Ably.'
+    }
+  }
+}
+
+rest.push.admin.publish(recipient, message)
+```
+
+```[php]
+$recipient = [  'transportType' => 'apns', 'deviceToken' => 'XXXXXXX' ];
+$data = [ 'push' =>
+          [ 'notification' =>
+            [ 'title' => 'Hello from Ably!',
+              'body' => 'Example push notification from Ably.'
+            ]
+          ]
+        ];
+$rest->push->admin->publish( $recipient, $data );
+```
+
+```[csharp]
+var recipient = new {
+    deviceId = "xxxxxxxxxxx"
+};
+var data = new {
+    notification = new {
+        title = "Hello from Ably!",
+        body = "Example push notification from Ably."
+    }
+};
+
+rest.push.admin.publish(recipient, data);
+```
+
+```[flutter]
+    'deviceId': 'xxxxxxxxxxx',
+};
+var data = {
+    'notification': {
+        'title': 'Hello from Ably!',
+        'body': 'Example push notification from Ably.',
+    },
+};
+
+rest.push.admin.publish(recipient, data);
+
+```
+
+h2(#payload-structure). Push notification payload structure
```

Comment:
Does this make sense to be further up the page considering the structure is required to publish correctly?